.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,106 @@
+name: 🐞 Bug report
```

Comment:
As part of this migration/clean up we need to also remove all of the usage of emoji characters in our templates.  They are not good for accessibility and should be removed from their usage here.  From looking through the changes, it looks like none of the emojis need replacing, and instead can just be removed.

This is true for all of the files in the change

Reply:
Looks like there are still some emojis around.